### PR TITLE
Dotnet sut fix

### PR DESCRIPTION
**Pull request by ZhangMan1126**
enable configuring with image version
dbclean for table with schema in ms SQL server for dotnet and java
updates tests for cleaning tables with specific schema

### CODE REVIEW

File: client-dotnet/src/EvoMaster.Controller.Tests/Controllers/db/DbCleanTestBase.cs [hunk 1]
```diff
@@ -68,24 +69,41 @@ public abstract class DbCleanTestBase {
...
 
-            SqlScriptRunner.ExecCommand(connection, "INSERT INTO Foo (x) VALUES (42)");
-            SqlScriptRunner.ExecCommand(connection, "INSERT INTO Bar (y) VALUES (42)");
+            SqlScriptRunner.ExecCommand(connection, "INSERT INTO "+getFooTable(type)+" (x) VALUES (42)");
+            SqlScriptRunner.ExecCommand(connection, "INSERT INTO "+getBarTable(type)+" (y) VALUES (42)");
+        }
+
+        private static string getFooTable(DatabaseType type){
+            switch (type){
+                case  DatabaseType.MS_SQL_SERVER: return "Foo.Foo";
```
Comment (arcuri82) [inline client-dotnet/src/EvoMaster.Controller.Tests/Controllers/db/DbCleanTestBase.cs:97]: add bit explanation on this

File: client-dotnet/src/EvoMaster.Controller.Tests/Controllers/db/DbCleanTestBase.cs [hunk 2]
```diff
@@ -68,24 +69,41 @@ public abstract class DbCleanTestBase {
...
+                default: return "Foo";
+            }
+        }
+        
+        private static string getBarTable(DatabaseType type){
+            switch (type){
+                case  DatabaseType.MS_SQL_SERVER: return "Foo.Bar";
+                default: return "Bar";
```
Comment (arcuri82) [inline client-dotnet/src/EvoMaster.Controller.Tests/Controllers/db/DbCleanTestBase.cs:105]: add bit explanation on this

File: client-dotnet/src/EvoMaster.Controller/Controllers/db/DbCleaner.cs [hunk 3]
```diff
@@ -171,11 +171,15 @@ public static class DbCleaner {
             command.ExecuteNonQuery();
         }
 
-        private static void DeleteTables(DbCommand command, string table, ISet<string> tableHasIdentify) {
-            command.CommandText = "DELETE FROM " + table;
+        private static void DeleteTables(DbCommand command, string table, string schmea, ISet<string> tableHasIdentify){
+            var tableWithSchema = table;
+            if (schmea.Length > 0 && !schmea.Equals(GetSchema(DatabaseType.MS_SQL_SERVER))){
+                tableWithSchema = schmea + "." + table;
```
Comment (arcuri82) [inline client-dotnet/src/EvoMaster.Controller/Controllers/db/DbCleaner.cs:177]: add bit explanation. is it that MSSQL always want the schema name specified?

**Review by arcuri82 — CHANGES_REQUESTED** [review]

File: client-dotnet/src/EvoMaster.Controller/Controllers/db/DbCleaner.cs (hunk 3 above)
Comment (arcuri82) [reply client-dotnet/src/EvoMaster.Controller/Controllers/db/DbCleaner.cs:177]: but this is not a test case, isn't it? so it is still unclear why MS_SQL_SERVER is treated specially here

File: client-dotnet/src/EvoMaster.Controller.Tests/Controllers/db/DbCleanTestBase.cs (hunk 1 above)
Comment (arcuri82) [reply client-dotnet/src/EvoMaster.Controller.Tests/Controllers/db/DbCleanTestBase.cs]: even if is a test, it is confusing at this point. add a comment like "The tests using MS_SQL_SERVER are defining a schema, called Foo"

**Review by arcuri82 — CHANGES_REQUESTED** [review]

File: client-dotnet/src/EvoMaster.Controller.Tests/Controllers/db/DbCleanTestBase.cs (hunk 1 above)
Comment (ZhangMan1126) [reply client-dotnet/src/EvoMaster.Controller.Tests/Controllers/db/DbCleanTestBase.cs]: this is added in a test `DbCleanerWithSchemaMsSqlServerTest`. please check methods `GetFooTable` and `GetBarTable` in the test

File: client-dotnet/src/EvoMaster.Controller/Controllers/db/DbCleaner.cs (hunk 3 above)
Comment (ZhangMan1126) [reply client-dotnet/src/EvoMaster.Controller/Controllers/db/DbCleaner.cs:182]: I thought the comment is for DbClearner tests. now, the comment is added in the latest commit.

**Pull request merged**